packages/skill-api/src/core/services/stripe-checkout.ts

```diff
@@ -155,22 +177,21 @@ export async function stripeCheckout({
           coupon: coupon.id,
         })
       } else if (merchantCoupon && merchantCoupon.identifier) {
-        // no ppp for bulk purchases
-        const isNotPPP = merchantCoupon.type !== 'ppp'
-        if (isNotPPP || quantity === 1) {
-          const {id} = await stripe.promotionCodes.create({
```

Comment:
I kept trying to reason through this conditional, but found it confusing. What I was understanding it distilled down to was, "if they are purchasing multiple seats and the coupon code is PPP, then don't apply that coupon." So I added some logic earlier up in the function that nulls-out the `merchantCoupon` variable if this is a multi-seat purchase. That simplifies things downstream.